### ISSUE TITLE
SourceMaps

### ISSUE THREAD

**Issue by GordonSmith** [phosphorjs/phosphor#274]
Would it be ok to ```sourceMap: true``` to each tsconfig.json file?

I am in the process of getting an acceptable src->tsc->webpack->debug workflow working in vscode and this is needed to play nice with the whole lerna / separate tests / webpack folder paradigm.

Example workflow:
* Open phosphor/packages/widgets in vs code
* press ctrl+shift+b to start a background ```tsc -w -p .```
* Externally call ```webpack -w``` on myApp
* Set some break points
* Press F5 to launch chrome debugging on myApp
* Hit break point / tweak code / refresh web page.

(If anyone has a better / more global workflow they could share that would be great!)

**Comment by sccolbert** [comment]
My typical workflow is similar to what you mention, run in browser until I hit a breakpoint, then tweak the relevant code. The JS produced by the TS compiler is clean enough that I don't really have a need for source maps, but I also dont have a problem adding a build command to produce them.

**Comment by GordonSmith** [comment]
This is with breakpoints set in vscode in the typescript?

**Comment by sccolbert** [comment]
No. Breakpoints set in Chrome dev tools. I use SublimeText as my editor, and I don't use the TS plugin on purpose. Not having rich auto-complete helps me keep my APIs small enough to fit in my head. I'm not suggesting you do this though :)

**Comment by GordonSmith** [comment]
Its a bit different when your on the other side of the fence (consuming lots of APIs).  I am quite envious of your minimal build environment and am now giving lerna a try.
